src/components/Notification/index.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Notification as NotificationItem } from '@santiment-network/ui'
+import SlideEntry from './Animated/SlideEntry'
+import styles from './Notification.module.scss'
+
+class Notification extends Component {
+  state = {
+    notifications: [],
+    dissmisedNotification: []
+  }
+
+  componentWillReceiveProps ({ notification }) {
+    if (notification) {
+      this.setState(
+        {
+          notifications: [...this.state.notifications, notification]
+        },
+        () => {
+          setTimeout(
+            () => this.closeNotification(notification.key),
+            notification.dismissAfter
+          )
+        }
+      )
+    } else {
+      this.setState({
+        notifications: []
+      })
+    }
+  }
+
+  closeNotification = notificationId => {
+    this.setState(
+      {
+        dissmisedNotification: [
+          ...this.state.dissmisedNotification,
+          notificationId
+        ]
```

Comment:
After every dismissed notification handling this garbage through the app lifetime